File: backup/Mybasic3/app/src/main/java/com/example/mybasic3/MainActivity.java
package com.example.mybasic3;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    private EditText e1;
    private Button b1;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        onclicklistener();
    }

    public void onclicklistener(){
        e1=(EditText)findViewById(R.id.editText);
        b1=(Button)findViewById(R.id.button);

        b1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(MainActivity.this,e1.getText(),Toast.LENGTH_LONG).show();
            }
        });
    }


}

File: android/kotlin part 4/Login/app/src/main/java/com/example/simplelogin/MainActivity.kt
package com.example.simplelogin

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var etUserName = findViewById<EditText>(R.id.et_user_name)
        var etPassword = findViewById<EditText>(R.id.et_password)
        var btnReset = findViewById<Button>(R.id.btn_reset)
        var btnSubmit = findViewById<Button>(R.id.btn_submit)

        btnReset.setOnClickListener{
            etUserName.setText("");
            etPassword.setText("");
        }

        btnSubmit.setOnClickListener{
            var userName = etUserName.text
            var password = etPassword.text

            val intent = Intent(this, TargetActivity::class.java)
            intent.putExtra("UserName", userName)
            intent.putExtra("Password", password)
            startActivity(intent)
        }
    }


}
